### DISCUSSION TITLE
Stardew Valley,but it cannot input the text

### DISCUSSION THREAD

**Discussion by JerryM33** [skyline-emu/skyline#323] [General]
![Screenshot_20220113_154207](https://user-images.githubusercontent.com/97395782/149287537-96aa1462-11c2-4f9a-9fdc-6e0c2b0ba67e.jpg)

**Comment by PixelyIon** [comment]
This is a known issue, @MCredstoner2004 is already working on implementing the `swkbd` (Software Keyboard) applet. It is going to take some time though due to the complexity.
